crates/rpc/rpc/Cargo.toml

```diff
@@ -84,6 +84,8 @@ jsonrpsee-types.workspace = true
 jsonrpsee = { workspace = true, features = ["client"] }
 
 [features]
+
```

Comment:
let's make js-tracer default here, for this pr, and solve making it non-default in this crate in a followup pr